code/_onclick/hud/fullscreen.dm

```diff
@@ -156,9 +156,11 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/fullscreen)
 	icon_state = "noise"
 
 /atom/movable/screen/fullscreen/high
-	icon = 'icons/hud/screen_gen.dmi'
-	screen_loc = "WEST,SOUTH to EAST,NORTH"
 	icon_state = "druggy"
+	alpha = 255
+	plane = LIGHTING_PLANE
+	layer = LIGHTING_ABOVE_ALL + 1 //Infinity plus one (not actually)
+	blend_mode = BLEND_MULTIPLY
```

Comment:
mild sus

Reply:
Blend multiply is a blend operation that allows this effect to happen. It multiplies the lighting rgb values by the ones present in the druggy hud overlay.

Reply:
yeah I take it back s fine